apps/frontend/src/pages/Build/BuildInfoDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { useParams } from "react-router-dom";
+import { BuildType } from "@/gql/graphql";
+
+import {
+  Dialog,
+  DialogBody,
+  DialogDismiss,
+  DialogFooter,
+  DialogText,
+  DialogTitle,
+  useDialogState,
+} from "@/ui/Dialog";
+import { Code } from "@/ui/Code";
+import { Link } from "@/ui/Link";
+
+const useDialogContent = (
+  buildType: BuildType | null | undefined,
+  referenceBranch: string | null | undefined,
+) => {
+  const { accountSlug, projectName } = useParams();
+
+  switch (buildType) {
+    case BuildType.Orphan:
+      return (
+        <>
+          <DialogText>
+            This build is <span className="font-semibold">orphan</span> because
+            there is no baseline for screenshots comparisons. To start visual
+            testing, create a{" "}
+            <span className="font-semibold">reference build</span> to get a
+            baseline and rebase your branch.
+          </DialogText>
+
+          <DialogText className="mt-1 mb-1">
+            <span className="font-semibold">
+              How to create a reference build?
+            </span>{" "}
+            Run Argos on reference branch
+            {referenceBranch ? (
```

Comment:
Type is wrong, I think we always have a reference branch according to server implementation.

Please fix types server side and remove the the condition.